website/src/user-guide/output-format.md

```diff
@@ -54,3 +54,5 @@ includes:
 parameters:
 	editorUrl: 'phpstorm://open?file=%%file%%&line=%%line%%'
 ```
+
+To make the text really clickable in your terminal, you might need to register the protocol in your system; see [eclemens/atom-url-handler](https://github.com/eclemens/atom-url-handler) for an example.
```

Comment:
`terminal, you` - no comma please

Reply:
AFAIK, comma "rules" are very liberal in English...